lib/create-scenarios.js

```diff
@@ -15,7 +17,8 @@ module.exports = function scenariosForTest(test) {
     const copy = Object.assign({}, test);
     copy.attrs = Object.assign({}, test.attrs);
     copy.attrs.description += ' (Strict Mode)';
-    copy.contents = `"use strict";\n${copy.contents}`;
+    copy.contents = usd + copy.contents;
```

Comment:
that's the best way to put USD in the code we make.

Reply:
:)